src/NativeMessageHost/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ if (FIREBREATH)
 endif()
 
 include_directories(
+    ${FB_GECKOSDK_SOURCE_DIR}
```

Comment:
This was to fix
#include "npapi.h" 
in NativeMessageHost/X11/PluginLoaderLin.cpp